frontend/src/util/worksheet_utils.js

```diff
@@ -302,41 +302,20 @@ export function createHandleRedirectFn(worksheetUuid) {
     };
 }
 
-export function getMinMaxKeys(item) {
-    if (!item) {
-        return { minKey: null, maxKey: null };
-    }
-    let minKey = null;
-    let maxKey = null;
+// Return the sort key at index subFocusIndex, if subFocusIndex is defined.
+// Otherwise, return the largest sort_key.
```

Comment:
The item here sounds more like a block (a bundle table block more than a bundle row)?

Reply:
Sure, but we use the `item` terminology all throughout the frontend code base -- probably should refactor this in a separate PR.